src/assets/data/markdown/ionic-weather-v4-step08.md

```diff
@@ -4,19 +4,40 @@ Cordova plugins are used when you want to access native functionality from your
 
 In this lab you will learn how to:
 
-- Install Cordova Plugins
-- Use Ionic Native wrappers
+- Install a Cordova plugin
+- Use an Ionic Native wrapper for a particular Cordova plugin
```

Comment:
Actually, the point is not to teach them how to install a Cordova plugin or how to use a particular wrapper, but to learn how to do it in general since there is a single common pattern that is followed for basically all of them.